tests/integration/tasks/test_cloud_for_customers.py

```diff
@@ -0,0 +1,24 @@
+from viadot.tasks import C4CToDF, C4CReportToDF
+from viadot.config import local_config
+
+
+def test_c4c_to_df():
+    credentials = local_config.get("CLOUD_FOR_CUSTOMERS")
+    credentials_prod = credentials["Prod"]
+    report_url = credentials_prod["server"]
```

Comment:
if this is a report too, why do we have another task for reports? Otherwise, if it's not a report, it shouldn't be called `report_url`.